src/pages/groupComparison/Survival.tsx

```diff
@@ -211,7 +211,11 @@ export default class Survival extends React.Component<ISurvivalProps, {}> {
                     survivalsByPrefixByAnalysisGroup,
                     survivalsByAnalysisGroup =>
                         _.mapValues(survivalsByAnalysisGroup, survivals =>
-                            survivals.sort((a, b) => a.months - b.months)
+                            _.orderBy(
+                                survivals,
+                                ['months', 'status'],
```

Comment:
if at all possible to avoid using text to represent the property names since i don't think it's typesafe.  we'd have to look at signature of orderBy to see if it accepts callback instead of his shorthand